guide/how-it-works/private-key-management/multi-key.md

```diff
@@ -74,7 +76,57 @@ A software wallet application or coordination software initiates a multi-sig wal
 
 {% include fact/close.html %}
 
-### Best practice
+### General user flow
+
+Alice, Robert, and Charles want to set up a multi-key wallet that has a total of 3 keys, with 2 keys required to send a transaction.
+
+In the first step, each of them independently generate a private key and a public key (technically an extended public key). They will use the private key for initiating and approving transactions, and will never share it it. The public key is used for construct the multi-key wallet for generating addresses to receive bitcoin to.
```

Comment:
Don't think the private key is required for "initiating" a transaction, only for signing/approving. Perhaps I misunderstand what you mean here...

Reply:
I don't see where it says that in the text...?